templates/users/reset_password.html

```diff
@@ -2,8 +2,11 @@
 
 {% block head %}
   
-<script defer>
+<script type="module">
+import { serverData } from "./static/js/modules/getServerData.js"
```

Comment:
I think this should be `serverData.js` instead of `getServerData.js`

Reply:
will fix